### ISSUE TITLE
Better pretty print in LambdaRunner

### ISSUE THREAD

**Issue by tomtdsis** [tdsis/lambda-forest#2]
LambdaRunner should have a better pretty print of the response body when executing ```LambdaRunner.run()```

**Comment by fssantana** [comment]
LambdaRunner will return an object that allows specific prints (pretty print):
- .print() - Prints the entire response: Response Body, Headers and Status Code
- .printBody() - Print Response Body
- .printHeaders() - Print Headers
- .printStatusCode() - Print Status Code

```java
public static void main(String [] args) {
        LambdaRunner.run("lambda-spec.json", LambdaHandler.class, args)
            .print()
            .printBody()
            .printHeaders()
            .printStatusCode();
}
```

**Comment by tommelo** [comment]
It looks great!

**Comment by fssantana** [comment]
Great! Closing issue.
Feature available in version 1.0.0

**Comment by tommelo** [comment]
@fssantana I think I didn't describe the issue that well, I just notice now...

The response body still being printed inline:
```
{
  "body" : "{\"name\":\"John\",\"age\":20,\"address\":\"Some street\"}",
  "headers" : { },
  "statusCode" : 200
}
```
A better pretty print would be:
```
{
  "body" : {
     "name":"John",
     "age":20,
     "address":"Some street"
  },
  "headers" : { },
  "statusCode" : 200
}
```

I guess just changing the type of the attribute ```body``` from ```String``` to ```Object``` in ```ResponseEntity``` should fix the pretty print. I'm not sure if changing the attribute type will affect the response flow with API Gateway.
